citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -363,6 +364,7 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
     			String href = citation.hasPreferredUrl() ? citation.getCustomUrl(citation.getPreferredUrlId()) : citation.getOpenurl();
     			
     			out.println("\t\t<td headers=\"details\">");
+    			out.println("\t\t\t<div class=\"detailsDiv\"><div class=\"titleDiv\"><a href=\"" + Validator.escapeHtml(href) + "\"><img id=\"" + citation.getCitationProperty("isnIdentifier") + "\" src=\"\" class=\"citationThumbnail\"></a><a href=\"" + Validator.escapeHtml(href) + "\" target=\"_blank\">" + Validator.escapeHtml( (String)citation.getCitationProperty( Schema.TITLE, true ) ) + "</a>");
```

Comment:
This will result in multiple elements on the page with the same ID if different parts of the same book are on the same reading list. This generally should be avoided as the ID should be unique. Using data-* attributes is the cleaner way.